holidays/countries/uzbekistan.py

```diff
@@ -9,51 +9,87 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-from holidays.groups import InternationalHolidays, IslamicHolidays
-from holidays.holiday_base import HolidayBase
+from datetime import date
+from gettext import gettext as tr
 
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import JAN, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC
+from holidays.groups import InternationalHolidays, IslamicHolidays, StaticHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SAT_SUN_TO_NEXT_WORKDAY
 
-class Uzbekistan(HolidayBase, InternationalHolidays, IslamicHolidays):
+
+class Uzbekistan(ObservedHolidayBase, InternationalHolidays, IslamicHolidays, StaticHolidays):
     """
-    https://www.officeholidays.com/countries/uzbekistan
+    References:
+        - https://en.wikipedia.org/wiki/Public_holidays_in_Uzbekistan
+        - `Labor Code 21.12.1995 <https://lex.uz/docs/-142859>`_
+        - `Labor Code 28.10.2022 <https://lex.uz/docs/-6257288>`_
     """
 
     country = "UZ"
+    default_language = "uz"
+    # Estimated label.
+    estimated_label = tr("%s* (*taxminiy)")
+    # %s (Observed).
+    observed_label = tr("%s (ko‘chirilgan)")
```

Comment:
Shouldn't we use ' instead of ‘ here?

Reply:
According to [Wiki](https://en.wikipedia.org/wiki/Uzbek_alphabet#Distinct_characters), the most correct symbol to use is `ʻ` (U+02BB), but official sites (including https://lex.uz) use `‘` (U+2018). That's why I used U+2018 as well.

Reply:
You're right, this is the correct one.